src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int NUMBER_COUNT = 100;
+    private Random random = new Random();
+    private ColorSupplier getRandomColor = new ColorSupplier();
+
+    public String getRandomBall() {
+        int randomNumber = random.nextInt(NUMBER_COUNT);
+        Ball ball = new Ball(getRandomColor.getRandomColor(), randomNumber);
+        return ball.toString();

```

Comment:
It's better to return Ball object from this method instead of String representation. Also don't create temporary variables if you can directly return a value from the method, as described in common mistakes.